README.md

```diff
@@ -8,6 +8,61 @@ Puppet configuration and environment management for the Grasshopper event engine
  * If you make changes to the backend code you will need to restart the app server. This can be done by ssh'ing into the client machine by running `service grasshopper restart`.
  * Even if you'd install all the components on your host OS, you would not be able to run the server as some of the npm modules are compiled during the provisioning step.
 
+### Dev Environment (e.g. on EC2)
+
+```
+# Provision an Ubuntu Trusty server
+# Then:
+local$ ssh devserver.ontheinternet
+devserver$ sudo apt-get update
+devserver$ sudo apt-get -y install git
+# (If you're testing puppet changes not in main master, don't forget to modify this next line!)
+devserver$ sudo git clone git://github.com/CUL-DigitalServices/grasshopper-puppet /opt/grasshopper-puppet
+devserver$ cd /opt/grasshopper-puppet
+
+# Edit common.json:
+# - to make tenant_hostname match your new server's hostname
+# - to change git config to match a tag if appropriate
+devserver$ sudo vim environments/dev/hiera/common.json
+
+# Copy some timetable data to import onto the server
+local$ scp timetabledata.json devserver.ontheinternet:/tmp/timetabledata.json
+
+# Back to the server to run puppet
+# This next step could take up to around 60mins if you have a slow server and lots of data to import!
+# Take note of the puppet command line displayed at the end, you may find it useful later
+devserver$ sudo ./provisioning/grasshopper/init.sh
```

Comment:
Can all steps prior to this be puppetised as well? It'd be nice to only have to run a single command.